pyproject.toml

```diff
@@ -95,11 +94,6 @@ omit = [
     "tests/test_tutorial/*/schemas/*",
 ]
 
-[tool.black]
-color = true
-line-length = 120
-target-version = ["py310"]
```

Comment:
Wouldn't this also affect blacken-docs? If so, we still need this!

Reply:
it's not affect for pre-commit blacken-docs